dags/main_summary.py

```diff
@@ -367,6 +367,8 @@
         arguments=[
             # restore legacy schema
             "--maps-from-entries",
+            "--partition-by",
+            "submission_date_s3",
```

Comment:
This field is already specified above in `static_partitions="submission_date_s3={{ds_nodash}}",`, but it also needs to appear here in order to defeat the default partitioning field (`submission_date`).